Testscripts/Linux/STOR-Lis-Disk.sh

```diff
@@ -76,6 +78,15 @@ done
 
 LogMsg "constants disk count = $disk_count"
 
+# Generation 2 VM does not support IDE or VHD format disk
+if [ -d /sys/firmware/efi ]; then
+    if [ $diskIDECount -ge 1 ] || [ $vhdFormat == 'vhd' ]; then 
```

Comment:
please check all cases use this bash STOR-Lis-Disk.sh, and if you don't define vhdFormat, here it will be empty?

Reply:
Test case will pass, but it will get error "unary operator expected", if use [[ ]], then no error message, will fix it.